packages/builder/src/pages/builder/app/[application]/settings/automations/index.svelte

```diff
@@ -35,9 +37,13 @@
   let timeRange = null
   let loaded = false
 
+  $: app = $apps.find(app => app.devId === $store.appId)
```

Comment:
might be worth doing `$store.appId.includes(app.appId)` as the automation section may need to handle both prod/dev apps soon!